apps/gcp/prow/pre/chatgpt/tasks.yaml

```diff
@@ -17,3 +17,20 @@ PingCAP-QE/test-infra:
       # - prompt_tpl: |
       #     This the context about context A: %s
       #   res_url: https://external.site/resource.html
+pingcap/docs: &docs
+  default: # default task profile, you can trigger it with `/review default`
+    name: default review summary
+    system_message: >
+      You are an experienced software developer. You will act as a reviewer for a GitHub Pull Request, and you should answer by markdown format.
+    user_prompt: |
+      Please follow the steps below to review:
+      1. Evaluate the title, description, and diff of the PR. If the title is unclear, suggest a concise and appropriate alternative. 
+      2. Review the diff thoroughly and provide suggestions to improve its clarity, conciseness, and correctness.
```

Comment:
```suggestion
      Please review the content as follows:
      1. Evaluate the title, description, and diff of the PR. If the title is unclear, suggest a concise and appropriate alternative.
      2. Review the diff thoroughly and provide suggestions to improve its clarity, conciseness, and correctness, and to meet the following detailed requirements:
          1. Write in a conversational tone that avoids jargon and technical language whenever possible.
          2. Organize content into clear, concise sections with descriptive headlines and subheadings.
          3. Use active voice and avoid passive verb constructions.
          4. Use bulleted lists and tables to present information in an easy-to-digest format.
          5. Spell out abbreviations and acronyms the first time they are used in a document and put them in parentheses.
          6. Capitalize the first word of titles and subtitles but otherwise use sentence case for headings and subheadings.
          7. Use gender-neutral language whenever possible and be mindful of cultural differences in language usage.
```